docs/dev/development.adoc

```diff
@@ -166,12 +166,14 @@ To release a new version on GitHub:
 
 There is a helper script link:/scripts/bump-version.sh[scripts/bump-version.sh] that changes version number in all the files listed above.
 
+Also, ensure you cut the version in the link:Changelog.md['Changelog.md'] i.e create a new version on top of current version.
+
```

Comment:
IIUC, this statement is asking the user to put the new version and its changelog at the top of the current version, right? If yes, then can we simplify the statement to just say so? The `ensure you cut the version` part is a little confusing to me. I am naive at theis release part, so if it makes sense to you, then I'm okay with keeping it :)

Reply:
Ok making it cleaner